Dockerfile

```diff
@@ -1,19 +1,26 @@
-# Teeny-tiny matplotlib image based on alpine
-FROM czentye/matplotlib-minimal:3.1.2
+FROM python:3.11-slim
 
 LABEL author="Phil Ewels" \
       description="MultiQC" \
-      maintainer="phil.ewels@scilifelab.se"
-
-RUN apk add --no-cache bash
+      maintainer="phil.ewels@seqera.io"
 
 # Add the MultiQC source files to the container
 ADD . /usr/src/multiqc
 WORKDIR /usr/src/multiqc
 
-# Install MultiQC
-RUN python -m pip install .
+# - Install `ps` for Nextflow
+# - Install MultiQC through pip
+# - Delete unnecessary Python files
+# - Remove MultiQC source directory
+# - Add custom group and user
+RUN apt-get update && apt-get install -y procps && rm -rf /var/lib/apt/lists/* && \
+    pip install -v --no-cache-dir . && \
+    find /usr/local/lib/python3.11 \( -iname '*.c' -o -iname '*.pxd' -o -iname '*.pyd' -o -iname '__pycache__' \) | xargs rm -r && \
```

Comment:
Is it possible to use a `find python3.11 (options) -exec rm -r {}\;` instead?

Reply:
I took this from a blog post which said this didn't work for the `__pycache__` dir for some reason. It was brief though and I didn't try it myself. I agree that it would be tidier so probably worth a go.

To be honest, this line only saves about 2MB so barely worth it 😅